doc/api/http.md

```diff
@@ -400,6 +400,14 @@ Emitted when the server sends a '100 Continue' HTTP response, usually because
 the request contained 'Expect: 100-continue'. This is an instruction that
 the client should send the request body.
 
+### Event: 'information'
+<!-- YAML
+added: v10.0.0
```

Comment:
This should be `REPLACEME`. It will be replaced upon release.

Reply:
Done.